.github/pull_request_template.md

```diff
@@ -0,0 +1,54 @@
+### Description of Changes
+
+(briefly outline the reason for changes, and describe what's been done)
+
+### Breaking Changes
+
+- None
+
+### Release Checklist
+
+Prepare:
+
+- [ ] Detail any breaking changes. Breaking changes require a new major version number, and a migration guide in wiki / README.md
+- [ ] Check `./gradlew assemble` passes w/ no errors
+
+Bump versions in:
+
+- [ ] CHANGELOG.md
+
+Integration tests:
+
+## T&T Only
+
+- [ ] Init SDK with only optimove tenant / config
+- [ ] Track events
+
+## Push Only
+
+- [ ] Init SDK with all credentials
+- [ ] Track events
+- [ ] Associate customer (verify both backends)
```

Comment:
Should this test be in T&T only too?

Reply:
(suppose in other variant it's just verify other backend? or do we need to verify both?)